visidata/_input.py

```diff
@@ -245,6 +245,8 @@ def find_nonword(s, a, b, incr):
             i += 1
 
         if i < 0: i = 0
+        # v may have a non-str type with no len()
+        v = str(v)
```

Comment:
Without this, we need `len(str(v))` in a lot more places!

VisiData casts `v` into a `str` in line 150, but then when `v` changes through the use of any of the other `editline()` commands, it can be changed to a non-str'. So, it needs to be re-cast. This is the cleanest way to not have `str()` peppered in a lot of other places.

Reply:
Do we need the rest of the `str(v)` changes then too?

Reply:
Yes, unfortunately. Because they involve a change in the value, before the value is `str()`.